src/components/SettingsButton/SettingsButton.js

```diff
@@ -1,6 +1,6 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
-import { Motion, spring } from 'react-motion';
+import { Spring, animated, interpolate } from 'react-spring';
```

Comment:
I took the opportunity to convert to React Spring. This is something I want to do everywhere (#233), because it's a nicer API and is more performant (it doesn't need to re-render on every frame!)